king_phisher/client/assistants/campaign.py

```diff
@@ -127,6 +153,37 @@ def __init__(self, application, campaign_id=None):
 			self.gobjects['label_intro_body'].set_text('This assistant will walk you through creating and configuring a new King Phisher campaign.')
 			self.gobjects['label_intro_title'].set_text('New Campaign')
 
+		self._url_selected = None
+		self._url_thread = None
+
+		self.domain_completion = Gtk.EntryCompletion()
+		self._domain_list_store = Gtk.ListStore(str)
+		self.domain_completion.set_model(self._domain_list_store)
+		self.domain_completion.set_text_column(0)
+		self.gobjects['entry_domain_filter'].set_completion(self.domain_completion)
+
+		self.kpm_target_file = None
+		self.kpm_dest_dir = None
+
+		tvm = managers.TreeViewManager(self.gobjects['treeview_url_selector'])
+		tvm.set_column_titles(
+			['Domain', 'Landng Page', 'Tags', 'URL'],
```

Comment:
Is the third entry tags or classifiers? Should try to keep the verbiage consistent.

Reply:
Yeah, this should be classifiers. Can you also please swap it with URL so it's last? Since the classifiers are so long, the URL is pushed off the screen and since that's what I want to set, it would be helpful if it were featured more predominately.